coderd/database/dbfake/dbfake.go

```diff
@@ -2480,6 +2489,8 @@ func (q *FakeQuerier) GetTemplateParameterInsights(ctx context.Context, arg data
 		}
 	}
 
+	// NOTE(mafredri): Add sorting if we decide on how to handle PostgreSQL collations.
```

Comment:
It will depend on the systems locale. Essentially if e.g. if `LC_ALL=C` or `LC_ALL=en_US.UTF-8` is set when calling `initdb`, it will produce different results.